crates/trycp_server/Cargo.toml

```diff
@@ -12,15 +12,21 @@ documentation = "https://github.com/holochain/holochain-rust"
 repository = "https://github.com/holochain/holochain-rust"
 
 [dependencies]
+in_stream = { version = "=0.0.44-alpha3", path = "../in_stream" }
 log = "0.4.8"
 env_logger = "0.7.0"
 structopt = "0.2.15"
 jsonrpc-core = "14.0"
 jsonrpc-ws-server = "14.0"
+jsonrpc-lite = "=0.5.0"
```

Comment:
is this used, or leftover from iterating?

Reply:
Also, why was it not used, instead implementing our own JsonRPC structs?

Reply:
I tried to use jsonrpc-lite and it didn't work over websockets, only over http, so that's why I didn't follow through.  Then I just took the sample work @neonphog did with jsonrpc for in_stream